src/actions/participants.js

```diff
@@ -43,4 +49,13 @@ const fileUpload = file => dispatch => {
     }
 };
 
-export { setRandomParticipant, removeParticipant, fileUpload, actions };
+const loadList = list => dispatch => {
+    let listOfParticipants = [];
+
+    if (list !== '') {
+        list.split(',').map(p => listOfParticipants.push({ name: p }));
```

Comment:
@filgaia can you do a trim() here, so in case the separated comma list have empty spaces we get just the name.